docs/source/user/conversion.rst

```diff
@@ -0,0 +1,97 @@
+.. _conversion:
+
+===============
+Data Conversion
+===============
+
```

Comment:
Please add these notes at top:

* Information in this page is implementation detail.  Normal PyMySQL users don't need this information.
* This is backdoor for low level problem.  Don't use this for (de)serializing application layer types.  Use ORM instead.